src/components/TableCells/JurisdictionReport.tsx

```diff
@@ -0,0 +1,26 @@
+import { DropDownCellProps } from '@onaio/drill-down-table';
```

Comment:
Intention is for this component to be used whenever we don't need anything fancy: either just a link or plain text